openrtb_ext/imp.go

```diff
@@ -14,9 +14,15 @@ type ExtImpPrebid struct {
 
 	// Bidder is the preferred approach for providing paramters to be interepreted by the bidder's adapter.
 	Bidder map[string]json.RawMessage `json:"bidder"`
+
+	Options *Options `json:"options,omitempty"`
 }
 
 // ExtStoredRequest defines the contract for bidrequest.imp[i].ext.prebid.storedrequest
 type ExtStoredRequest struct {
 	ID string `json:"id"`
 }
+
+type Options struct {
+	Echovideoattrs bool `json:"echovideoattrs"`
```

Comment:
Nitpick: The variable name can have proper capitalization and naming, ie: `EchoVideoAttributes `

Reply:
Sure, renamed.